next.config.js

```diff
@@ -16,6 +16,12 @@ module.exports = {
         destination: "https://discord.gg/WHpCrPqeqx",
         permanent: true,
       },
+      {
+        source: "/linkedin-page",
+        destination:
+          "https://www.linkedin.com/company/hawaiians-in-technology/",
+        permanent: true,
```

Comment:
Interesting, we might want to set this to `permanent: false`. I'd expect our LinkedIn redirect to change soon (say, with the ongoing Page vs Group convo).

Getting me to look into nextjs redirects, I think we adjust this to the Discord redirect (line 17) to be false too. Since it expires after awhile.

Reply:
https://nextjs.org/docs/pages/api-reference/next-config-js/redirects

Reply:
sounds good, making the update for both discord and linkedin from `permanent: true` to `permanent: false`